webapp/client/src/pages/TaxNumberPage.js

```diff
@@ -275,7 +275,7 @@ export default function TaxNumberPage({
   );
 
   const requestNewTaxNumberCheckbox = (
-    <FormRowCentered key="requestNewTaxNumberRow">
+    <>
```

Comment:
Why did you remove the FormRowCentered?

Reply:
With the new `flex: 1` on the sub-elements of `FormRowCentered` the items within (heading, text and checkbox) were treated as columns and put next to each other (as items within a row are supposed to) That's why I removed it. They never should have been in a row anyways.
Oh, but is it bad that I removed the key? Is that needed for anyting?

Reply:
Ok, but could you re-add the key?